src/test/java/dev/streamx/cli/ingestion/publish/PublishPayloadCommandTest.java

```diff
@@ -88,20 +90,38 @@ public void shouldRejectInvalidFile(QuarkusMainLauncher launcher) {
     assertThat(result.getErrorOutput()).contains("Replacement is not recognised as valid JSON.");
   }
 
-  @Test
-  public void shouldRejectNonExistingJsonPath(QuarkusMainLauncher launcher) {
+  @ParameterizedTest
+  @ValueSource(strings = {
+      "content[0].bytes",
+      "$..bytes",
+      "*.bytes",
+  })
+  public void shouldRejectNonExistingJsonPath(String path, QuarkusMainLauncher launcher) {
```

Comment:
I believe that we should add all non existing values/replacements to payload instead of throwing an exception.